src/gluon/events/packages/ApplicationProdRequested.ts

```diff
@@ -76,6 +79,9 @@ export class ApplicationProdRequested extends BaseQMEvent implements HandleEvent
         try {
             const project = applicationProdRequestedEvent.project;
 
+            const qmProject = await this.gluonService.projects.gluonProjectFromProjectName(project.name);
```

Comment:
```suggestion
            const qmProject: QMProject = await this.gluonService.projects.gluonProjectFromProjectName(project.name);
```

Will probably need to add the correct import to implement this above change ^

Reply:
Done tx